scripts/buildDockerfile.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+docker build . $ADDITIONAL_DOCKER_BUILD_ARGS --file Dockerfile --tag local/clang-format-action:latest
```

Comment:
ShellCheck complains about the unquoted expansion of this variable. They [recommend](https://github.com/koalaman/shellcheck/wiki/SC2086) using arrays instead, but since we are using environment variables for this purpose here, that [doesn't work](https://unix.stackexchange.com/a/393096/203826).

Thus, I don't see a way to fix this

Reply:
could we...

* hardcode the `-v` flag since that doesn't change, and quote the variable expansion for the workspace variable passed in from `action.yml`?
* directly reference the workspace variable in the script? you might have to use the format `$GITHUB_WORKSPACE` instead of the `${}` format: 
```
docker build . -v "$GITHUB_WORKSPACE":/clang-format-action/workspace --file Dockerfile --tag local/clang-format-action:latest
```

I noticed you pushing a lot of commits while working on the draft PR, so it's entirely possible you tried these - let me know if you've already exhausted those options 😅

Reply:
yeah, I think that should be possible. It removes the possibility of specifying any other option though. However, this doesn't seem to be a problem as of now...